tests/basics/string_fstring.py

```diff
@@ -65,3 +65,9 @@ def foo(a, b):
 # Still allow ! in expressions.
 print(f"{'1' if a != '456' else '0'!r:8s}")
 print(f"{'1' if a != '456' else '0'!s:8s}")
+
+# Concatenation of adjacent f-strings.
+print(f"" f"")
+print(f"a" f"b")
+print(f"{x}" f"{y}")
+print(f"a{x}b" f"cd" f"e{y}f")
```

Comment:
I would add one additional test to this. 

```python
print(
    f"a{x}b" 
    f"cd" 
    f"e{y}f"
)
```

This is a useful thing to do if wanting to keep code pep8 compliant, under the 80 character width for a line.
Want to make sure that the lexer properly handles the line breaks as well.

Reply:
Good idea.  I've now done that (although slightly different to the way you suggested).

Reply:
I thought it could possibly be a failure mode if something went screwy with the lexer.. Can't hurt to have a test for it.

question on the tests.. Does it check what actually gets output to make sure nothing got truncated or added during the concatenation?

Reply:
> Does it check what actually gets output to make sure nothing got truncated or added during the concatenation?

Yes.  It runs the same test with CPython and makes sure the output from that matches the output from MicroPython (unless there's a corresponding .py.exp file which is used as the expected output instead of running CPython).